### ISSUE TITLE
rusty-tags doesn't seem to work with cargo workspaces

### ISSUE THREAD

**Issue by jrmuizel** [dan-t/rusty-tags#31]
rusty-tags tries to open Cargo.lock in the crate instead of the workspace

**Comment by dan-t** [comment]
Can you please check if `rusty-tags 1.1.1` fixes it?

**Comment by dan-t** [comment]
Some feedback would be nice.

**Comment by jrmuizel** [comment]
I get the Fetching source of dependencies ...

```
Couldn't find source string in package: '{"dependencies": Array([String("app_units 0.3.0 (registry+https://github.com/rust-lang/crates.io-index)"), String("byteorder 0.5.3 (registry+https://github.com/rust-lang/crates.io-index)"), String("core-graphics 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)"), String("dwrote 0.1.4 (registry+https://github.com/rust-lang/crates.io-index)"), String("euclid 0.10.3 (registry+https://github.com/rust-lang/crates.io-index)"), String("gleam 0.2.30 (registry+https://github.com/rust-lang/crates.io-index)"), String("heapsize 0.3.7 (registry+https://github.com/rust-lang/crates.io-index)"), String("ipc-channel 0.5.1 (registry+https://github.com/rust-lang/crates.io-index)"), String("offscreen_gl_context 0.5.0 (registry+https://github.com/rust-lang/crates.io-index)"), String("serde 0.8.19 (registry+https://github.com/rust-lang/crates.io-index)"), String("serde_codegen 0.8.19 (registry+https://github.com/rust-lang/crates.io-index)"), String("serde_derive 0.8.19 (registry+https://github.com/rust-lang/crates.io-index)")]), "name": String("webrender_traits"), "version": String("0.11.0")}'!
```

When running rusty-tags 1.1.1 in the sample directory of https://github.com/servo/webrender

**Comment by jrmuizel** [comment]
Actually I was running rusty-tags 1.3.0

**Comment by dan-t** [comment]
The problem should be fixed with `rusty-tags 2.0.0`.

**Comment by jrmuizel** [comment]
Everything seems to work great now. Thanks.
